src/hammer-tech/hammer_tech.py

```diff
@@ -164,6 +173,37 @@ def library_from_json(json: str) -> Library:
     return Library.from_json(json)
 
 
+# Struct that holds an extra library and possible prefix.
+class ExtraLibrary(NamedTuple('ExtraLibrary', [
+    ('prefix', Optional[PathPrefix]),
+    ('library', Library)
+])):
+    __slots__ = ()
+
+    def to_setting(self) -> dict:
+        raise NotImplementedError("No clean implementation for JSON export of library yet")
```

Comment:
Is this something we want and should be an issue?

Reply:
It's not really important right now. Fixing this is hard since it requires wrangling with/replacing python_jsonschema_objects